public/app/plugins/panel/datagrid/DataGridPanel.tsx

```diff
@@ -208,6 +214,10 @@ export function DataGridPanel({ options, data, id, fieldConfig, width, height }:
     return <PanelDataErrorView panelId={id} fieldConfig={fieldConfig} data={data} />;
   }
 
+  if (!isDatagridEnabled()) {
+    return <PanelDataErrorView panelId={id} message="Datagrid is not enabled" fieldConfig={fieldConfig} data={data} />;
+  }
+
```

Comment:
I feel like this is an elegant way of having the panel behind a ff. I don't know how it didn't hit me earlier :)) Thank you!